pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -92,6 +91,10 @@ def __init__(
         self.fullname = f"{self.name}-{self.version}"
         self.build_args = build_args
 
+        self.library_install_prefix = (
+            Path(build_dir).resolve() if build_dir else self.pkg_root.parent
```

Comment:
Does `build_dir` refer to `PYODIDE_ROOT/packages`? That is surprising to me, maybe we should rename it?

Reply:
Good point. This should be fixed.